editor/editor_node.cpp

```diff
@@ -4839,6 +4839,7 @@ EditorNode::EditorNode() {
 	VBoxContainer *center_vb = memnew(VBoxContainer);
 	main_hsplit->add_child(center_vb);
 	center_vb->set_h_size_flags(Control::SIZE_EXPAND_FILL);
+	center_vb->set_custom_minimum_size(Size2(700, 0) * EDSCALE);
```

Comment:
I don't think this is the proper fix, that's too arbitrary.

Reply:
It's the width of the profiler. What size do you recommend?